src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+const ul = document.querySelector('ul');
+
+function convertToNumber(text) {
+  return +(text.replace('$', '').replace(',', ''));
+}
+
+const sortList = (input) => {
+  return [...input].sort((item1, item2) => {
+    if (convertToNumber(item1.dataset.salary)
+      === convertToNumber(item2.dataset.salary)) {
+      return item1.innerText.localeCompare(item2.innerText);
+    }
```

Comment:
i believe you don't need this check. just sort values as numbers

Reply:
but without this check the order of the employees does not meet the requirements of the task.